sw/host/provisioning/cp_lib/src/lib.rs

```diff
@@ -129,7 +120,13 @@ pub fn run_sram_cp_provision(
     )?;
 
     // Inject provisioning data into the device.
-    provisioning_data.send(spi_console)?;
+    data_in.send(spi_console)?;
+
+    // Wait to receive CP device ID.
+    let _ = UartConsole::wait_for(spi_console, r"Exporting CP device ID ...", timeout)?;
+    let _data_out = ManufCpProvisioningDataOut::recv(spi_console, timeout, true)?;
```

Comment:
@cfrantz @moidx We should update this to log (via JSON) the CP portion of the device ID so the orchestrator script can pick this up and update it with the FT portion (https://docs.google.com/document/d/1zVgZjLPMiBQhainWnyFjJx_fTnYx-8HpDJwLQHhltyo/edit?usp=sharing)

Reply:
to match #25387

Reply:
How about something like this:
```
#[derive(Debug, Clone, Default, Serialize)]
struct ChipProbeResponse {
  pub device_id: String,
  // can put other stats in here if needed.
}

let mut response = ChipProbeResponse::default();
response.device_id = data_out
        .device_id
        .iter()
        .map(|v| format!("{v:08X}"))
        .collect::<Vec<String>>()
        .join("");

let doc = serde_json::to_string(&response)?;
println!("CHIP_PROBE_DATA: {doc}");
```

Reply:
Ah yes, thanks! updated. @moidx @cfrantz just a heads up the CP device ID will be logged as a little-endian string. I can update it if you prefer parsing a big-endian string, just let me know.

Reply:
Lets transform the logged data to whichever format is like the command-line argument to the `ft` harness (which, btw, is the same form the FT logged data uses, and uses the same conversion code above).

Reply:
ah ok, I think that is big-endian format: https://cs.opensource.google/opentitan/opentitan/+/master:sw/host/provisioning/util_lib/src/lib.rs;l=20?q=hex_string_to_u32_arrayvec&ss=opentitan%2Fopentitan

I updated the PR to log the CP device ID response in big-endian.